state/accountsDBApi.go

```diff
@@ -0,0 +1,221 @@
+package state
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	chainData "github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go/common"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
```

Comment:
Where is this used in this function?

Reply:
because the esdt storage handler expects `vmcommon.UserAccountHandler` and I have to cast `state.UserAccountHandler` to `vmcommon.UserAccountHandler`